server/routes/column.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  getColumns,
+  createColumn,
+  getColumn,
+  updateColumn,
+  reorderColumn,
+} = require("../controllers/column");
+
+router.use(protect);
+
+router.route("/").get(getColumns);
+router.route("/").post(createColumn);
+router.route("/:id").get(getColumn);
+router.route("/:id").patch(updateColumn);
+router.route("/:id").patch(reorderColumn);
```

Comment:
Need a `delete` column method + route; also, I like the difference with the `patch` request by separating full methods. Well done!